pkg/kubelet/server/stats/summary.go

```diff
@@ -283,12 +284,27 @@ func (sb *summaryBuilder) containerInfoV2ToStats(
 			PageFaults:      &pageFaults,
 			MajorPageFaults: &majorPageFaults,
 		}
+		// availableBytes = memory  limit (if known) - workingset
+		if !isMemoryUnlimited(info.Spec.Memory.Limit) {
+			availableBytes := info.Spec.Memory.Limit - cstat.Memory.WorkingSet
```

Comment:
Does memory limit include swap space? If not, couldn't `availableBytes` end up being negative?